Gruntfile.js

```diff
@@ -3,6 +3,7 @@
 /* globals Set */
 var webpackConfig = require( './webpack.config' );
 var installChanged = require( 'install-changed' );
+var json2php = require( 'json2php' );
```

Comment:
This needs to be installed as dependency. It works now because `json2php` is available by way of `@wordpress/dependency-extraction-webpack-plugin`. I only noticed this because there is also an issue with the version of `json2php` installed there (0.0.4) [not supporting booleans](https://github.com/daniel-zahariev/json2php/pull/9) and a number of blocks are broken because of that.

Reply:
Thanks, added the dependency 👍 
Right now I added v0.0.4 here, because I think if we up the version it should be done consistently and not use different versions here